Assets/MixedRealityToolkit/Interfaces/SceneSystem/IMixedRealitySceneSystemEditor.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.SceneSystem;
+
+/// <summary>
+/// Optional editor-only interface for use with facade inspectors.
+/// If a scene system service does not implement this interface, the facade will not be rendered.
+/// </summary>
+public interface IMixedRealitySceneSystemEditor
```

Comment:
>IMixedRealitySceneSystemEditor [](start = 17, length = 30)

This interface is in the global namespace, please move to the non-global namespace (I think this is also probably what failed the build)